### PR TITLE
update lesson example, discuss shadowing, show IntelliJ generation

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,202 @@
-# Accessor and Mutator Methods
+# Scope
```
Comment (kcleland0818) [inline README.md]: I think this is supposed to be about accessor and mutator methods - not scope?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,202 @@
...
+In Java, we define public **accessor** and **mutator** methods to get and set private instance variables.
+
+- An accessor method takes no parameters and returns the value of a private instance variable.
+  The method return type matches the instance variable's declared type.
+  Accessor methods are also called **getter** methods.
+    - An accessor method for a non-boolean instance variable `abc` would be named `getAbc`.
+    - An accessor method for a boolean instance variable `abc` would be named `isAbc`.
+- A mutator method takes one parameter whose type and name match the instance variable's declared type
+  and name. Mutator methods don't return a value and are also called **setter** methods.
+    - A mutator method for instance variable `abc` would be named `setAbc` and would take a parameter named `abc`.
```
Comment (kcleland0818) [inline README.md]: "A mutator method for instance variable `abc`..."

Add an "an" between "for" and "instance"

File: README.md [hunk 3]
```diff
@@ -1,32 +1,202 @@
...
+
+Let's see how this works with the `Dog` class, which has instance variables `name` and `wagginTail`.
```
Comment (kcleland0818) [inline README.md]: Typo in `wagginTail`: should be `waggingTail`

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: Yes, thanks!

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (linda-seiter) [reply README.md]: fixed

**Review by developerjaym — APPROVED** [review]

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**